docs/UserGuide.md

```diff
@@ -17,14 +17,18 @@ ReadyBakey is a **desktop app that manages orders and customer contact informati
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `readybakey.jar` from [here](https://github.com/AY2122S2-CS2103-F09-4/tp/releases).
+2. Download the latest `readybakey.jar` from [here](https://github.com/AY2122S2-CS2103-F09-4/tp/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your ReadyBakey.
+3. Copy the file to the folder you want to use as the _home folder_ for your ReadyBakey.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+4. Open a command window in that folder
+
+5. Run the command `java -jar readybakey.jar`  (i.e., run the command in the same folder as the jar file) 
```

Comment:
Do you want to just write that this is specific to both windows and mac users? Can also put the ability for windows users to just double click the jar?

Reply:
I don't think there is a need to specify it works for both windows and mac users because this is the appropriate way to open a jar file across all OS. Since it works for every OS, I don't think I need to specify it works for any specific OS.